MilitaryManager.Attachments.API/Controllers/AttachmentsController.cs

```diff
@@ -88,7 +88,7 @@ public string GenerateNewDocument()
                 _logger.LogError(ex, $"Template for {templateName} document is not available");
             }
 
-            var unit = _unitService.GetUnitAsync(5).Result;
+            var unit = _unitService.GetUnitsByIdAsync(5).Result;
```

Comment:
Use await, remove constant id and .Result

Reply:
it's not about this task at all).